docs/adr/0010-common-auth-for-applications.md

```diff
@@ -0,0 +1,93 @@
+# A common authentication for applications
+
+- Status: accepted
+- Deciders: @tumido, @HumairAK, @anishasthana, @4n4nd
+- Date: 2021-01-19
+
+Technical Story: [description | ticket/issue URL] <!-- optional -->
+
+## Context and Problem Statement
+
+Applications deployed within Operate First are run on top of OpenShift. Many applications require users to log in in order to be able to enforce RBAC, quotas and sandboxing. It is also essential for auditing. User management of the applications should be unified so the same user can access all the applications with a single set of credentials as well as the user ensured his username stays unique and constant across the applications. The accepted solution should provide SSO, so the user can cary over the identity across different applications. The identity and authentication should be based on the OpenShift authentication.
```

Comment:
> as well as the user ensured his username stays unique and constant across the applications.

Not quire sure I understand -- What do you mean by 'username stays unique and constant' ?

Reply:
Ah nvm, I think I understand. 

Let's rephrase: 
> User management of the applications should be unified so the same user can access all the applications with a single set of credentials as well as the user ensured his username stays unique and constant across the applications. 

to: 

> User management of the applications should be unified so the same user can access all the applications with a single set of credentials. The username should be unique, but also utilized across applications.

Reply:
I like that. I'll use your wording. :+1: